books/free-programming-books-hi.md

```diff
@@ -39,6 +39,7 @@
 ### Java
 
 * [Java \| Hindi](https://www.learnhindituts.com/java) - LearnHindiTuts.com
+* [Java \| Hindi](https://ehindistudy.com/2022/10/14/java-pdf-book-hindi/) - ehindistudy.com
```

Comment:
I'm a little suspicious of this since the PDF is screenshots of a mobile phone. Curious if this is truly offered for free.

Reply:
Well Hindi tech guides are scarce in nature. 
If any source is available, I think it can be deemed necessary to place it here.